src/Button/button.css

```diff
@@ -262,12 +265,43 @@
 
         /* Use hover color for dropdown section of the dropdownbutton */
         &--raised:nth-child(2):focus {
-            border-color: rgba(234, 244, 255, 0.4);
+            border: 1px solid rgba(234, 244, 255, 0.4);
             background-color: rgba(234, 244, 255, 0.4);
         }
         &--primary:nth-child(2):focus {
-            border-color: #033976;
+            border: 1px solid #033976;
             background-color: #033976;
         }
     }
 }
+/* IE11 hacks */
+@media all and (-ms-high-contrast: none) {
+    /* Graceful degradation: remove the rounded corners on all buttons apart from circle.
+       These are not rendered correctly in IE11, where they produce white pixels in the corners  */
+    .d2ui-button {
+        border-radius: 0;
```

Comment:
If you disable this line and open the example app in IE11 you will see some white pixels in the rounded corners. Maybe you know of a way to get rid of them.